third-party/realdds/doc/DDS Use cases.md

```diff
@@ -0,0 +1,255 @@
+
+realdds is a library to help control a realsense camera over network using librealsense API.
+It is a work in progress aimed at, ultimately, enabling remote users to do with the camera everything a user can do with a USB connected camera, and more (multiple clients for frames, etc...)
+
+realdds uses DDS topics to send and receive information to/from the camera. See https://github.com/IntelRealSense/librealsense/blob/dds/doc/DDS/DDS%20ICD.md
+
+Users can use realdds stand-alone, or use librealsense with the cmake `BUILD_WITH_DDS` flag on, to use DDS devices seemlessly as any other librealsense device.
+
+# Actors
+
+## Server/Camera
+
+The **server** or **camera** is either a physical device with built in capabilities supporting these use cases, like Intel RealSense D585, or a dedicated software operating some other device, e.g., Intel RealSense D455, giving it the required capabilities.
+
+An example software tool like this is the [dds-server](https://github.com/IntelRealSense/librealsense/tree/dds/tools/dds/dds-server)
+
+## User/Client
+
+The **client** is the end **user** of the camera. The **user** consumes the data generated by the camera. The **user** can activate or stop camera data streaming and control various options like camera exposure time or brightness.
```

Comment:
All these bolds are a little confusing for me (especially when all the "user"s are bold later))

Reply:
I use bold when mentioning an `actor` in the document.
I think it highlights the action doer in the use case steps, but it is cumbersome to write and to read without a proper md reader. After approval GitHub will show it in a friendlier manner.

Reply:
ok